lib/secp256k1/point.ex

```diff
@@ -21,20 +21,48 @@ defmodule Bitcoinex.Secp256k1.Point do
                   :erlang.is_map_key(:z, term)
 
   @doc """
-  serialize_public_key serializes a compressed public key
+  parse_public_key parses a public key
   """
-  @spec serialize_public_key(t()) :: String.t()
-  def serialize_public_key(%__MODULE__{x: x, y: y}) do
+  # @spec parse_public_key(sec) :: %__MODULE__.t()
+  def parse_public_key(<<0x04, x::binary-size(32), y::binary-size(32)>>) do
+    %__MODULE__{x: :binary.decode_unsigned(x), y: :binary.decode_unsigned(y)}
+  end
+
+  def parse_public_key(<<prefix::binary-size(1), x_bytes::binary-size(32)>>) do
```

Comment:
Worth adding a comment that matches on compressed public keys while the one above will match for uncompressed public keys.

Reply:
accepted.